src/pages/api/connect.js

```diff
@@ -0,0 +1,21 @@
+import {dbConnect} from '../../dbConnect';
+import Item from '../../models/itemModel';
+
+export default async function handler(request, response) {
+	await dbConnect();
+
+	if (request.method === 'POST') {
+		const data = JSON.parse(request.body);
+
+		await Item.create(data);
+
+		response.status(200).json({
+			message: 'item created',
+		});
```

Comment:
And here is the addition for the `addItems` Zustand function.

```suggestion
	if (request.method === 'POST') {
		const data = JSON.parse(request.body);

		await Item.create(data);
		const items = await Item.find();

		response.status(200).json({
			message: 'item created',
			items
		});
```